guides/doc-Provisioning_Guide/topics/Networking.adoc

```diff
@@ -211,6 +211,36 @@ To configure network services on {Project}'s integrated {SmartProxy}, complete t
 # hammer proxy info --name "_{foreman-example-com}_"
 ----
 
+[[Configuring_Networking-Multiple_DHCP_Subnets_via_Puppet]]
+==== Multiple DHCP Subnets via Puppet
+
+The installer have options available only for a single DHCP subnet. One way to define more than one subnet is via Puppet, the tool that the installer utilizes to perform the configuration.
+
+For every additional subnet, create an entry in `/etc/foreman-installer/custom-hiera.yaml` file:
+
+----
+dhcp::pools:
+ example.com:
+   network: 192.168.99.0
+   mask: 255.255.255.0
+   gateway: 192.168.99.1
+   range: 192.168.99.5 192.168.99.49
+----
+
+Execute {foreman-installer} to perform the changes and verify that the `/etc/dhcp/dhcpd.conf` contains appropriate entries. Subnets must be then defined in {Project} database.
+
+[[Configuring_Networking-Multiple_DHCP_Subnets_Manually]]
+==== Multiple DHCP Subnets Manually
+
+DHCP server configuration can be also managed manually, or with an external configuration tool. In that case, re-run the installer with a switch that disables management of the DHCP configuration completely while keeping the DHCP feature turned on.
+
+[options="nowrap" subs="+quotes,attributes"]
+----
+# foreman-installer --foreman-proxy-dhcp true --foreman-proxy-dhcp-managed false
+----
+
+From now on, DHCP configuration (e.g. `/etc/dhcp/dhcpd.conf`) is completely out of {foreman-installer} and Puppet control. {Project} will only connect to the DHCP server via OMAPI to perform DHCP record management.
```

Comment:
IMHO we shouldn't mention Puppet explicitly. That's just an implementation detail of the installer.

Reply:
I removed this paragraph completely.